src/ada/base/non_phyical_objects.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
nice! I did not know about this. I am all for removing the " from any typechecked references :)